Stylesheets/Components/input.css

```diff
@@ -0,0 +1,75 @@
+label {
+    display: block;
+    margin-bottom: 2px;
+}
+
+.input-container {
+    margin: 3rem auto;
+    width: fit-content;
+    height: fit-content;
+    background-color: var(--st-background-grey);
+    padding: 1rem;
+    border: 2px solid grey;
+}
+
+.input-group {
+    width: 50vh;
+}
+
+.input-group input {
+    background: #fff;
+    border: 1px solid #999;
+    padding: 0.6rem 0.7rem;
+    font-size: .8rem;
+    margin: 5px 0;
+    width: 100%;
+    outline: none;
+    border-radius: 2px;
+}
+
+.err {
+    color: var(--st-span-text);
+}
+
+#error-ip {
```

Comment:
try to be consistent with naming. you are using error and err both for denoting error.

Reply:
Thanks for pointing it out. I have made the changes